src/Npgsql/Npgsql.csproj

```diff
@@ -18,8 +18,9 @@
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetFramework)' == 'net461' OR '$(TargetFramework)' == 'netstandard2.0' ">
     <PackageReference Include="System.Memory" Version="4.5.3" />
-    <PackageReference Include="System.Threading.Tasks.Extensions" />
+    <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.3" />
```

Comment:
We should move all these versions out to Directory.Build.targets

Reply:
Maybe not here? It's a fix PR, not refactoring.